src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -610,6 +619,28 @@ public static DynamoModel Start(IStartConfiguration configuration)
         // Token representing the standard library directory
         internal static readonly string StandardLibraryToken = @"%StandardLibrary%";
 
+
+        private void InitializeLuceneConfig()
+        {
+            addedFields = new List<string>();
```

Comment:
Skip these during TestMode I guess

Reply:
If we skip indexing during tests, how do we want to handle the search tests?

Reply:
I was thinking creating dedicated tests for index and search in a single test or try to make it work when running in parallel? The current setup will not work for sure, unfortunately..

Reply:
yes, I checked that there are 7 calls to the SearchViewModel.Search in the LibraryTests.
An option could be adding another Search method in SearchViewModel that will be calling the SearchDictionary.Search (as previously) or try to call the SearchDictionary.Search directly in the tests.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/bf77f33d-f54f-47a8-8351-e74addac1cbc)